apps/common/utils/verify_code.py

```diff
@@ -78,7 +82,7 @@ def __generate(self):
         return code
 
     def __send_with_sms(self):
-        send_sms_async.apply_async(args=(self.target, self.code), priority=100)
+        send_sms_async.apply_async(args=(self.target, self.code, self.user_info), priority=100)
 
     def __send_with_email(self):
         subject = self.other_args.get('subject', '')
```

Comment:
There is a lack of information about this section. Please rephrase your question so that I can review it accurately.